### PR TITLE
Update italian.json5

### PR DESCRIPTION
**Pull request by conky77**

### CODE REVIEW

File: translation/italian.json5 [hunk 1]
```diff
@@ -342,15 +342,15 @@
...
-        "SCHEMA_STRING_DOESNT_MATCH_PATTERN": "la stringa '{0}' non corrisponde alla regex '{1}'",
+        "SCHEMA_OBJECT_MISSING_REQUIRED_KEY": "object missing \"richiede\" la chiave {0}",
+        "SCHEMA_FALSE_SCHEMA": "lo schema è &quot;false&quot;, quindi nulla supererà la convalida",
+        "SCHEMA_STRING_DOESNT_MATCH_PATTERN": "la stringa '{0}' non corrisponde alla Espressione Regolare '{1}'",
         "SCHEMA_RECURSION_LIMIT_REACHED": "la convalida ha una profondità massima di 128",
-        "SCHEMA_NUMBER_LESS_THAN_MIN": "numero {0} inferiore al minimo {1}",
-        "SCHEMA_NUMBER_GREATER_THAN_MAX": "numero {0} maggiore del massimo {1}",
-        "SCHEMA_NUMBER_LESSEQ_EXCLUSIVE_MIN": "numero {0} inferiore o uguale al minimo esclusivo {1}",
-        "SCHEMA_NUMBER_GREATEREQ_EXCLUSIVE_MAX": "numero {0} maggiore o uguale al massimo esclusivo {1}",
-        "SCHEMA_STRING_TOO_LONG": "la stringa {0} era maggiore di maxLength {1}",
```
Comment (molsonkiko) [inline translation/italian.json5:353]: `maxLength` and `minLength` are also JSON schema keywords; that's why I didn't translate them

File: translation/italian.json5 [hunk 2]
```diff
@@ -260,52 +260,52 @@
...
+        "JSON5_HEX_NUM": "I numeri esadecimali sono consentiti solo in JSON5",
+        "JSON5_NUM_LEADING_DECIMAL_POINT": "I numeri con un punto decimale iniziale sono consentiti solo in JSON5",
         "JSON5_COMMA_AFTER_LAST_ELEMENT_ARRAY": "Virgola dopo l'ultimo elemento dell'array",
         "JSON5_COMMA_AFTER_LAST_ELEMENT_OBJECT": "Virgola dopo l'ultima coppia chiave-valore dell'oggetto",
-        "JSON5_NUM_TRAILING_DECIMAL_POINT": "I numeri con un punto decimale finale sono solo una parte di JSON5",
+        "JSON5_NUM_TRAILING_DECIMAL_POINT": "I numeri con un punto decimale finale sono consenti solo in JSON5",
         // ============== BAD messages ==============
         "BAD_UNTERMINATED_MULTILINE_COMMENT": "Commento su più righe senza terminazione",
-        "BAD_PYTHON_COMMENT": "I commenti '#' in stile Python non fanno parte di alcuna specifica JSON ben accettata",
-        "BAD_STRING_CONTAINS_NEWLINE": "La stringa letterale contiene una nuova riga",
```
Comment (molsonkiko) [inline translation/italian.json5:283]: I assume `"a capo"` is the way to say `newline` in Italian?

In any case, make sure all instances of `&quot;` are converted to `\"`.

Comment (conky77) [reply translation/italian.json5:283]: @molsonkiko  in my opinion "a capo"  is the best translation. 
It is the short form of  "ritorno a capo" that is similar to "carriage return" (e.g. #13#10) 
https://context.reverso.net/traduzione/italiano-inglese/ritorno+a+capo
It is used for "carriage return", newline, wordwrap
"nuova riga" is the translation of "new line", but not for "newline"

**Comment by molsonkiko** [comment]
I went through the changes you made in this PR, and I implemented all the ones I believed were appropriate in [my latest commit](https://github.com/molsonkiko/JsonToolsNppPlugin/commit/66a81e3517bfb6439b17e43f6f052951b71bf535). I am therefore closing this PR. Thank you so much for your help, @conky77 !

Next I'm going to start working on [translating message boxes](https://github.com/molsonkiko/JsonToolsNppPlugin/blob/66a81e3517bfb6439b17e43f6f052951b71bf535/translation/italian.json5#L395). I'll let you know when I've come up with my first draft of the translation.

**Pull request closed without merging**